integration/test/e2e/e2e_test.go

```diff
@@ -45,6 +45,7 @@ func TestE2e(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	CheckImageExistOrFail(t, r, "retagger-e2e", "2.6")
 	CheckImageExistOrFail(t, r, "retagger-e2e", "3.7")
```

Comment:
It might be time to turn these into a [table test](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)